CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+# Contributing to OpenShift documentation
```

Comment:
Having a contributing guide in this repo would be useful, but it seems odd to have one that just talks about contributing to documentation and points people over to the documentation repo.

I'd at least structure it so that the top level entry is more generic, e..g:

# Contributing to OpenShift Components

blah blah welcome new person, here's a guide to help you.....

## Contributing to docs
## Contributing to tests
## Contributing to console
## Contributing to designs/feature proposals

(i'm not asking you to populate all those sections, just that we structure this so that those sections can easily be added in the future and encourages people to add them)

Also note that this repository (origin) is almost entirely just test code at this point, the rest of the project is maintained in various independent repos.  That said, it's still probably an appropriate starting point or a lot of people

Reply:
Gotcha, implemented the recommendations. Thanks for the feedback.